mlx/ops.cpp

```diff
@@ -2767,6 +2768,22 @@ inline void run_conv_checks(const array& in, const array& wt, int n_dim) {
         << " input: " << in.shape() << " and weight: " << wt.shape();
```

Comment:
If the group>1, then the last dimension of weight * group == the last dimension of input shape